src/everest/detached/jobs/everserver.py

```diff
@@ -308,26 +169,51 @@ def main():
         return
 
     try:
+        # wait until the api server is running
+        is_running = False
+        while not is_running:
+            try:
+                requests.get(url + "/", verify=cert, auth=auth, proxies=PROXY)
+                is_running = True
+            except:
+                time.sleep(1)
+
         update_everserver_status(status_path, ServerStatus.running)
 
-        run_model = EverestRunModel.create(
-            config,
-            simulation_callback=partial(_sim_monitor, shared_data=shared_data),
-            optimization_callback=partial(_opt_monitor, shared_data=shared_data),
+        response = requests.post(
+            url + "/" + START_ENDPOINT, verify=cert, auth=auth, proxies=PROXY
```

Comment:
Did you consider starting the experiment here: https://github.com/equinor/ert/blob/6c86e6aeb4e0610945156e6a845f145e0227f296/src/everest/bin/everest_script.py#L132, where we start the server initially? Then passing the config through here.

Reply:
What is the difference between `everserver.py` and `everest_script.py `?

Reply:
Something like:
```python
@app.post("/experiments/")
async def submit_experiment(experiment: EverestConfig, ...):
```

Reply:
Whats the advantage of doing this?

Reply:
We force more separation by starting the experiment through an api instead of relying on files on disk. We would also like to extend this concept to ert, and having a clear separation helps then.

Reply:
> What is the difference between everserver.py and everest_script.py ?
`everserver.py` is the actual server, while `everest_script` is what starts the server and queries for status

Reply:
So the current client does:

```
submit_server_to_cluster(path_to_config_file)
wait_for_server
monitor
```

think in the future we would like:

```
connect_to_already_running_server
requests.post(server_url, json=everest_config)
monitor
```

Then we are one step closer to having the experiment server, and more clear separation between the frontend and the backend.